pkg/controllers/scheduling/schedule.go

```diff
@@ -103,11 +171,6 @@ func bind(
 	clusterClient clusterclient.Interface,
 	placementDecisionLister clusterlisterv1alpha1.PlacementDecisionLister,
 ) error {
-	// sort clusterdecisions by cluster name
```

Comment:
sorting by name may still have value since it will stabilize the order as relative scoring changes.  It will also keep consumers from thinking the order is about priority of scoring which may be less stable over time.